pkg/proxy/node_test.go

```diff
@@ -146,6 +151,70 @@ func TestNodePodCIDRHandlerUpdate(t *testing.T) {
 	}
 }
 
+func TestNodeManagerUpsert(t *testing.T) {
+	oldKlogOsExit := klog.OsExit
+	defer func() {
+		klog.OsExit = oldKlogOsExit
+	}()
+	klog.OsExit = customExit
```

Comment:
uh.... I don't like that. How about having a `NodeManager.exit` that the unit tests can override instead?

oh, lol, you're just copying existing tests. So that makes it _definitely_ not-parallel-execution-safe.